MapEditorReborn/API/Extensions/SomeEnumHelperMethodsThatMakeDoingWhatYouWantEasier.cs

```diff
@@ -0,0 +1,154 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MapEditorReborn.API.Extensions
+{
+    public static class SomeEnumHelperMethodsThatMakeDoingWhatYouWantEasier
```

Comment:
Rename it to something normal

Reply:
Lol that was what I found it as online. Will do